### DISCUSSION TITLE
mutateFormDataBeforeCreate function in V2.x

### DISCUSSION THREAD

**Discussion by zbadshah** [filamentphp/filament#8249] [Help]
Please help as I used this function before and it works. 

However, now I am using mutateFormDataBeforeCreate function in a new program which updates $data. 

I confirmed it with dd($data) and the fields are updated. However, when the record is created, the updated fields are null. 

I confirmed it with dd($this->data) in beforeCreate() function, which shows null values. 

What could cause this?

**Comment by dissto** [comment]
Checked your models `$fillable` ?

**Reply by zbadshah** [reply]
Yes, of course. I saw earlier discussions and got this idea. Checked. All seems fine. The field names were re-checked Also in resource and fillable etc.
